### DISCUSSION TITLE
Q3 Iteration Plans

### DISCUSSION THREAD

**Discussion by WilliaJing** [guardstrikelab/carla_apollo_bridge#116] [Show and tell]
# Q3 Iteration Plans

## New version update

 * [x] 🏃Complete Apollo upgrade to v8.0.0
 * [x] 🏃Replace the latest image file with CyberRT

## Project code refactoring

 * [x] Reduce project deployment steps
 * [x] As a module of Apollo, bridge is integrated into Apollo

## Map

 * [x] 🏃 Convert all of Carla's maps to Apollo maps

## Perception

 * [x] [Traffic light module] Supports identification of traffic lights, enabling vehicles to start and stop according to rules

## Planning

* [x] Support to set the start point and end point at the same time in Dreamviewer
## Control

Our team will continue to add updates，Welcome to discuss with us

**Comment by Michael-Fuu** [comment]
Hi, I have question about integrating bridge into Apollo, it that mean we only need a apollo container and carla container? can we still spawn objects in carla by python scripts?

**Reply by WilliaJing** [reply]
yes,yes,We will release this integration code soon.

**Comment by XiaoFei9704** [comment]
We just released a new Release that implements all the features of the Q3 Plan：
https://github.com/orgs/guardstrikelab/discussions/135
